benchmarks/src/test/java/io/grpc/benchmarks/driver/LoadWorkerTest.java

```diff
@@ -62,6 +63,18 @@ public void setup() throws Exception {
     marksQueue = new LinkedBlockingQueue<>();
   }
 
+  @After
+  public void tearDown() throws Exception {
```

Comment:
```suggestion
  public void tearDown() {
```

Don't think we need throws here.